src/effects/effectsmanager.cpp

```diff
@@ -70,81 +70,106 @@ const QSet<ChannelHandleAndGroup>& EffectsManager::registeredChannels() const {
     return m_pEffectChainManager->registeredChannels();
 }
 
-const QList<QString> EffectsManager::getAvailableEffects() const {
-    QList<QString> availableEffects;
+bool alphabetizeEffectNameIdPairs(const QPair<QString, QString>& pair1,
```

Comment:
I think I'll refactor this so EffectManager keeps a QList<EffectManifest> sorted alphabetically. But yeah, I don't understand what the issue with QPair is.